src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -46,6 +46,7 @@ public ErrorResponse handlePropertyReferenceException(
     }
 
     @ExceptionHandler({
+            RegistrationException.class,
             DataIntegrityViolationException.class,
```

Comment:
I would not say that this is a good idea to handle here a raw SQL exception like `DataIntegrityViolationException`, I'd rather prevent it from happening by checking ISBN uniqueness in the service

Reply:
I have such an ISBN and user check for uniqueness. I've added the handling of this error in case another inconsistency occurs